packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -160,10 +166,12 @@ export function setValueForProperty(
     let attributeValue;
     if (type === BOOLEAN || (type === OVERLOADED_BOOLEAN && value === true)) {
       attributeValue = '';
+    } else if (__DEV__) {
+      attributeValue = stringifyWithPerformanceWarning(value);
     } else {
       // `setAttribute` with objects becomes only `[object]` in IE8/9,
       // ('' + value) makes it output the correct toString()-value.
-      attributeValue = '' + (value: any);
+      attributeValue = (value: any).toString();
```

Comment:
Why was this changed? The comment above no longer reflects the actual code. Aren't we concerned about IE8/9 anymore? If not, then the comment should also be updated, I guess.

Reply:
https://reactjs.org/blog/2016/01/12/discontinuing-ie8-support.html

Just IE9+ it seems

Reply:
@Swieckowski please revert his back as well.